resolvers/unfinishItem.js

```diff
@@ -0,0 +1,146 @@
+const gql = String.raw;
+
+const {getUserId} = require('../utils');
+const {NotFoundError} = require('../errors');
+const {sendMetric} = require('../stats');
+
+const unfinishItem = async (parent, {id, token}, ctx) => {
+	const fragment = gql`
+		fragment ItemWithQuoteAndProject on Item {
```

Comment:
Pourquoi y'a besoin d'autant de truc dans le fragment ? genre canceledReminders ?

Reply:
I had planned to put back the reminders in place but I guess I didn't. Should it be done?